Description: Replace JS script links to CDN with local copies
 Embedding JS files from commercial CDNs is - with a reason - considered
 a privacy breach for documents in Debian packages. This patch replaces
 the links to JS scripts at maxcdn.com with local links.
Forwarded: https://gitlab.com/mailman/postorius/issues/222
Author: Jonas Meurer <mejo@debian.org>
Last-Update: 2017-10-02

---
 src/postorius/templates/postorius/base.html |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/src/postorius/templates/postorius/base.html
+++ b/src/postorius/templates/postorius/base.html
@@ -16,8 +16,8 @@
     <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
     <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
     <!--[if lt IE 9]>
-      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
-      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+      <script src="{% static 'postorius/libs/html5shiv.min.js' %}"></script>
+      <script src="{% static 'postorius/libs/respond.min.js' %}"></script>
     <![endif]-->
 </head>
 <body>
